Adafruit_BBIO/sysfs.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2014 MIT OpenCourseWare
```

Comment:
Needs a proper copyright assignment with the right owner. As it stands now, it's a boiler plate assignment.